demo/index.html

```diff
@@ -45,6 +45,8 @@
 
       <!-- Component ---------------------------------------------------------->
       <px-demo-component>
+        <input id="multiSelect" class="btn btn--primary" type="button" value="Open Multi Select"
```

Comment:
Rather than adding this button, i'd recommend using the `props` values to change the context browser when the `Multiselect` config item is chosen. I'm happy to go over strategies for that in person if you'd like.